src/swap/components/SwapProvider.tsx

```diff
@@ -58,6 +58,13 @@ export function SwapProvider({
   const { from, to } = useFromTo(address);
   const { sendTransactionAsync } = useSendTransaction(); // Sending the transaction (and approval, if applicable)
 
```

Comment:
Why are you adding those spaces in line 60 and 64?

Spaces have to be tactical to devide group of code, in this case those Hooks are part of the `// Core Hooks`.

Reply:
Looks like it was an oversight on my end. will fix